hr_expense_petty_cash/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module add Petty Cash on Paid By choice.
```

Comment:
This, I believe, needs more explanation. What the module does specifically and why.